<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>178</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>ON</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>OFF</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>LED 1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>178</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="pushButton_7">
      <property name="text">
       <string>ON</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_8">
      <property name="text">
       <string>OFF</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>LED 2</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>105</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Melody</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>181</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="pushButton_9">
      <property name="text">
       <string>ON</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_12">
      <property name="text">
       <string>OFF</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>260</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Distance</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>320</y>
     <width>178</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="pushButton_10">
      <property name="text">
       <string>ON</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_11">
      <property name="text">
       <string>OFF</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>201</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLCDNumber" name="lcdNumber"/>
    </item>
    <item>
     <widget class="QDial" name="dial">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>180</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="horizontalSlider">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>180</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>UP</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>DOWN</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Piboto [Goog]&quot;;</string>
   </property>
   <property name="text">
    <string>Moter</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>slot_on()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>slot_off()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>slot_on2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>477</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>slot_off2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>606</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>560</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>slot_exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>slot_piezo_on()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_10</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>slot_ultra_on()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>slot_moter_up()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>slot_moter_down()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>576</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>slot_moterbar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>slot_moterdial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_12</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>slot_piezo_off()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_11</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>slot_ultra_off()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>561</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>555</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_on()</slot>
  <slot>slot_off()</slot>
  <slot>slot_exit()</slot>
  <slot>slot_on2()</slot>
  <slot>slot_off2()</slot>
  <slot>slot_piezo_on()</slot>
  <slot>slot_ultra_on()</slot>
  <slot>slot_ultra_off()</slot>
  <slot>slot_moter_up()</slot>
  <slot>slot_moter_down()</slot>
  <slot>slot_moterbar()</slot>
  <slot>slot_moterdial()</slot>
  <slot>slot_piezo_off()</slot>
 </slots>
</ui>
